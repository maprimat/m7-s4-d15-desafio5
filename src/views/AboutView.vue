<template>
    <div class="about">
        <h1>Vista About</h1>
    </div>
</template>

<script>
export default {
    name: 'AboutView'
}
</script>

<style scoped lang="scss"></style>