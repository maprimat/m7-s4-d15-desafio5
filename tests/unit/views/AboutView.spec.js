import AboutView from "@/views/AboutView.vue";
import { shallowMount } from "@vue/test-utils";

describe("AboutView", () => {
    test("Se valida snapshot de estructura HTML: AboutView", () => {

        //Selección el Sujeto de pruebas
        const wrapper = shallowMount(AboutView);

        //Aserción
        expect(wrapper.html()).toMatchSnapshot(`
        <div class="about">
            <h1>Vista About</h1>
        </div>
    `);
    });
});
