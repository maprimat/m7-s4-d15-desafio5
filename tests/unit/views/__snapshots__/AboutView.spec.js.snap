// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView Se valida snapshot de estructura HTML: AboutView: 
        <div class="about">
            <h1>Vista About</h1>
        </div>
     1`] = `
<div class="about">
  <h1>Vista About</h1>
</div>
`;
